### DISCUSSION TITLE
Utilisation du script pour ender 3 V3 KE / Using script for ender 3 V3 KE

### DISCUSSION THREAD

**Discussion by BUZZ-SEB** [Guilouz/Creality-Helper-Script-Wiki#580] [Issues]
Bonjour, le script est il utilisable avec la ender 3 KE firmware V1.1.0.14?
j'ai utiliser la manip de CREALITY https://github.com/CrealityOfficial/Ender-3_V3_KE_Annex/tree/main pour installer fuild pour connecter ORCASLICER  mais te temps d'impression restant ne s'affiche pas. c'est du a ORCASLICER ou au Fluid  du site CREALITY?
merci pour votre aide.

Hello, can the script be used with ender 3 KE firmware V1.1.0.14?
I used the CREALITY manipulation https://github.com/CrealityOfficial/Ender-3_V3_KE_Annex/tree/main to install fluid to connect ORCASLICER but the remaining print time is not displayed. Is it due to ORCASLICER or to the Fluidd of the CREALITY site?
Thanks for your help.

![screenshot](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/170262312/b37f5837-53ab-4656-bcea-4e6a2c197814)

**Comment by Guilouz** [comment]
Oui il est compatible avec les KE.
